app/views/matches/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= I18n.t('activerecord.models.match') %></h1>
+<p>
+    <% if Time.zone.now.after?(@match.kickoff_at) %>
```

Comment:
Move logic to model? DOM structure may eventually depend on view, but `Match#display_score` already has some display logic.